feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go

```diff
@@ -100,12 +100,22 @@ func TestStandbyControllerCardReboot(t *testing.T) {
 	}
 
 	t.Logf("rebootSubComponentRequest: %v", rebootSubComponentRequest)
+	startReboot := time.Now()
 	rebootResponse, err := gnoiClient.System().Reboot(context.Background(), rebootSubComponentRequest)
 	if err != nil {
 		t.Fatalf("Failed to perform component reboot with unexpected err: %v", err)
 	}
 	t.Logf("gnoiClient.System().Reboot() response: %v, err: %v", rebootResponse, err)
 
+	watch := dut.Telemetry().Component(rpStandby).RedundantRole().Watch(
+		t, 10*time.Minute, func(val *telemetry.QualifiedE_PlatformTypes_ComponentRedundantRole) bool {
+			return val.IsPresent()
```

Comment:
are you checking for if the value exist, or if the value is the one expected?

Reply:
Check if the value exists/IsPresent.